waspc/src/ExternalCode.hs

```diff
@@ -4,20 +4,23 @@ module ExternalCode
     , fileAbsPath
     , fileText
     , readFiles
+    , SourceExternalCodeDir
     ) where
 
 import qualified Data.Text.Lazy as TextL
 import qualified Data.Text.Lazy.IO as TextL.IO
 import Data.Text (Text)
-import qualified Path
-import qualified Path.Aliases as Path
 
 import qualified Util.IO
+import StrongPath (Path, Abs, Rel, Dir, (</>))
+import qualified StrongPath as SP
 
+-- | External code directory in Wasp source, from which external code files are read.
+data SourceExternalCodeDir
```

Comment:
Yes! I wanted to avoid mixing it with GeneratedExternalCodeDir, so I named this one SourceExternalCodeDir. It is not perfect name but I think it is still better than the ones you suggested, plus there is comment.